aw/ViewControllers/Reaches/RunListTableViewCell.swift

```diff
@@ -2,20 +2,111 @@ import CoreData
 import UIKit
 
 class RunListTableViewCell: UITableViewCell, MOCViewControllerType {
-    @IBOutlet weak var conditionColorView: UIView!
-    @IBOutlet weak var riverName: UILabel!
-    @IBOutlet weak var sectionLabel: UILabel!
-    @IBOutlet weak var difficultyLabel: UILabel!
-    @IBOutlet weak var favoriteButton: UIButton!
-    @IBOutlet weak var distanceLabel: UILabel!
 
     var managedObjectContext: NSManagedObjectContext?
 
     var reach: Reach?
 
-    override func awakeFromNib() {
-        super.awakeFromNib()
-        // Initialization code
+    private let conditionColorView: UIView = {
```

Comment:
Just fyi, I know I suggested doing all the layout in code. What I've done in the past is put the view elements in the storyboard or nib, then do all the constraints and coloring in code. But constructing views in code works as well, the main thing is not to split styling between storyboard and code, because it gets really hard to figure out what's coming from where (as we saw with the left bar color on the run cell).